src/sentry/api/helpers/group_index/update.py

```diff
@@ -738,3 +719,30 @@ def update_groups(
         result["inbox"] = inbox
 
     return Response(result)
+
+
+def handle_is_subscribed(
+    is_subscribed: Any,
```

Comment:
```suggestion
    is_subscribed: bool,
```

This function only gets called if we know `is_subscribed` is a boolean.